I'm sorry, but an uncaught exception occurred.

While loading <'Image' u'images/heat.png'>:
  File "game/script.rpy", line 151, in script
    scene heat with Dissolve(5.0)
IOError: Couldn't find file 'images/heat.png'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 151, in script
    scene heat with Dissolve(5.0)
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/ast.py", line 1397, in execute
    renpy.exports.with_statement(trans, paired)
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/exports.py", line 1646, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/display/core.py", line 2610, in do_with
    clear=clear)
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/display/core.py", line 3101, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs)
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/display/core.py", line 3611, in interact_core
    self.draw_screen(root_widget, fullscreen_video, (not fullscreen_video) or video_frame_drawn)
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/display/core.py", line 2441, in draw_screen
    renpy.config.screen_height,
  File "render.pyx", line 492, in renpy.display.render.render_screen
  File "render.pyx", line 263, in renpy.display.render.render
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/display/layout.py", line 777, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 263, in renpy.display.render.render
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/display/transition.py", line 368, in render
    top = render(self.new_widget, width, height, st, at)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 263, in renpy.display.render.render
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/display/layout.py", line 777, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 263, in renpy.display.render.render
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/display/layout.py", line 777, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 263, in renpy.display.render.render
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/display/transform.py", line 933, in render
    return transform_render(self, width, height, st, at)
  File "accelerator.pyx", line 116, in renpy.display.accelerator.transform_render
  File "render.pyx", line 263, in renpy.display.render.render
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/display/image.py", line 501, in render
    return wrap_render(self.target, width, height, st, at)
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/display/image.py", line 307, in wrap_render
    rend = render(child, w, h, st, at)
  File "render.pyx", line 170, in renpy.display.render.render
  File "render.pyx", line 263, in renpy.display.render.render
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/display/im.py", line 607, in render
    return cache.get(self, render=True)
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/display/im.py", line 271, in get
    surf = image.load()
  File "/home/the_g00se_/Загрузки/renpy-7.4.4-sdk/renpy/display/im.py", line 688, in load
    raise e
IOError: Couldn't find file 'images/heat.png'.

Linux-5.8.0-50-generic-x86_64-with-debian-bullseye-sid
Ren'Py 7.4.4.1439
Я не буду искать оправданий 1.0
Thu Apr 29 23:43:07 2021
